.github/actions/setup-rust/action.yml

```diff
@@ -3,7 +3,7 @@ description: Setup Rust with specified toolchain, target and components.
 inputs:
   toolchain:
     description: 'Toolchain'
-    default: 1.58.1
+    default: 1.60.0
```

Comment:
technically, msrv is still 1.58.1, just need 1.60.0 or nightly to use -C coverage

Reply:
We probably still want to test it compiles against 1.58.1 still, so we might want a separate default version only when checking for code coverage.